examples/rest-angular/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "rest-angular",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "update": "npx msw init public",
```

Comment:
This script won't work, as you try to init a worker in the `public` directory, while your worker is in `src/mockServiceWorker.js`. I assume Angular respects some JSON configuration and copies certain files into the build public directory. 

Please, either replace this with `npx msw init src`, or move `src/mockServiceWorker.js` into a `public` directory. Thanks!